components/slider/index.css

```diff
@@ -366,52 +400,55 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-Slider-value {
-  margin-inline-start: var(--spectrum-slider-label-gap-x);
+  margin-inline-start: var(--mod-slider-label-margin-start, var(--spectrum-slider-label-margin-start));
 }
 
+/* Slider misses core-tokens for tick variant */
 .spectrum-Slider-ticks {
   display: flex;
   justify-content: space-between;
 
   z-index: 0;
 
-  margin: 0 var(--spectrum-slider-track-margin-offset);
-  margin-block-start: calc(
-    var(--spectrum-slider-tick-mark-height) +
-    1px
-  );
+  margin-inline: var(--mod-slider-track-margin-offset, var(--spectrum-slider-track-margin-offset)) ;
+  margin-block-start: calc(var(--spectrum-slider-height-size) / 2 - var(--spectrum-slider-tick-mark-height) / 2);
```

Comment:
Line 414 and line 501 appear to be causing the ticks version to have extra space from the label to the controls. Visual:

![slider-ticks-label-distance](https://user-images.githubusercontent.com/25614178/216721421-0ad601f0-2fca-40da-9cc0-aea63d54b442.png)